app/models/concerns/foreman/thread_session.rb

```diff
@@ -69,10 +69,15 @@ def current=(o)
             raise(ArgumentError, "Unable to set current User, expected class '#{self}', got #{o.inspect}")
           end
 
-          if o.is_a?(User)
-            Rails.logger.info "Current user: #{o.login} (#{o.admin? ? 'administrator' : 'regular user'})"
+          Rails.logger.debug?
+            if o.is_a?(User)
```

Comment:
how about checking something like `if o.is_a?(User) && !o.hidden?`? this would prevent logging the system accounts.

Reply:
In such case, please also change the log message, otherwise we'll get report why some users are logged as info and why some as debug. Especially when we'll have syslog in place.